index.html

```diff
@@ -49,5 +49,191 @@ <h1 id="name">
             </ul>
         </div>
     </section>
+
+    <section id="work-section">
+        <div class="portfolio">
```

Comment:
- [ ] As I mentioned in the style.css file the usage of the grid for the main container, I think that you dont need to take project cards in another `<div class="portfolio">`. so can give display grid to the section which is the main containers of all cards and then, you can apply flex in the individuals' cards as you are already doing.